[2024-02-10T02:51:20.314+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: censoEnade.extract_zip_file manual__2024-02-10T02:50:41.306526+00:00 [queued]>
[2024-02-10T02:51:20.326+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: censoEnade.extract_zip_file manual__2024-02-10T02:50:41.306526+00:00 [queued]>
[2024-02-10T02:51:20.327+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-10T02:51:20.345+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_zip_file> on 2024-02-10 02:50:41.306526+00:00
[2024-02-10T02:51:20.353+0000] {standard_task_runner.py:60} INFO - Started process 483 to run task
[2024-02-10T02:51:20.357+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'censoEnade', 'extract_zip_file', 'manual__2024-02-10T02:50:41.306526+00:00', '--job-id', '181', '--raw', '--subdir', 'DAGS_FOLDER/censoEnade.py', '--cfg-path', '/tmp/tmp3_j3qyga']
[2024-02-10T02:51:20.360+0000] {standard_task_runner.py:88} INFO - Job 181: Subtask extract_zip_file
[2024-02-10T02:51:20.415+0000] {task_command.py:423} INFO - Running <TaskInstance: censoEnade.extract_zip_file manual__2024-02-10T02:50:41.306526+00:00 [running]> on host 7e5b5e202339
[2024-02-10T02:51:20.516+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='NÃ©fi Fernandes' AIRFLOW_CTX_DAG_ID='censoEnade' AIRFLOW_CTX_TASK_ID='extract_zip_file' AIRFLOW_CTX_EXECUTION_DATE='2024-02-10T02:50:41.306526+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-10T02:50:41.306526+00:00'
[2024-02-10T02:51:22.401+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T02:51:22.413+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=censoEnade, task_id=extract_zip_file, execution_date=20240210T025041, start_date=20240210T025120, end_date=20240210T025122
[2024-02-10T02:51:22.540+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T02:51:22.568+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
